pages/mylist.js

```diff
@@ -0,0 +1,21 @@
+import { getMyList } from 'server/services/myListClient';
+import MyListPage from 'components/Pages/MyListPage';
+
+export default function MyList({ moviesData }) {
+  return <MyListPage moviesData={moviesData} />;
+}
+
+export async function getServerSideProps({ req }) {
+  try {
+    const myList = await getMyList({ req });
+
+    return {
+      props: {
+        moviesData: myList ? myList.results : [],
+      },
+    };
+  } catch (e) {
+    console.error(e);
+    return { props: {} };
+  }
+}
```

Comment:
Try with these changes:
```suggestion
export async function getServerSideProps({ req }) {
  try {
    const myList = await getMyList(req);

    return {
      props: {
        moviesData: myList ? myList : [],
      },
    };
  } catch (e) {
    console.error(e);
    return { props: {} };
  }
}
```